Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -239,11 +252,36 @@ def track_workspace_artist(self, workspace, artists, data_replace_cb=None,
                 def data_replace_cb(_, __):
                     logger.warning("Updating data on this plot type is not yet supported")
             artist_info = self.tracked_workspaces.setdefault(name, [])
+
             artist_info.append(_WorkspaceArtists(artists, data_replace_cb,
+                                                 is_normalized,
                                                  spec_num))
-
+            self.check_axes_distribution_consistency()
         return artists
 
+    @staticmethod
+    def _on_off_to_bool(on_or_off):
```

Comment:
I can't find any uses of this method in mantid?

Reply:
Good catch! I'd forgotten to remove it